libcxxabi/src/cxa_exception.cpp

```diff
@@ -589,6 +589,11 @@ void __cxa_end_catch() {
     }
 }
 
+void __cxa_call_terminate(void* unwind_arg) throw() {
```

Comment:
We need to add a libc++abi test for this function. Don't forget `XFAIL: using-built-library-before-llvm-19`.